SalsaCompiler/SalsaCompiler/main.swift

```diff
@@ -52,80 +52,89 @@ let exportFile: URL = {
 // Find a name for the working directory
 let workingDirectory: String = {
   while true {
-    let directory = "/tmp/Salsa/\(UUID().uuidString)"
+    let directory = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(UUID().uuidString, isDirectory: true).absoluteString
```

Comment:
Nit: might be helpful to extract `URL(fileURLWithPath: NSTemporaryDirectory())` into its own variable (could even be outside the while loop!) since this line has gotten long.

Thanks filesystem apis...